lib/kubernetes-deploy/runner.rb

```diff
@@ -104,7 +105,9 @@ def run(verify_result: true, allow_protected_ns: false, prune: true)
 
       if deploy_has_priority_resources?(resources)
         @logger.phase_heading("Predeploying priority resources")
-        predeploy_priority_resources(resources)
+        ::StatsD.measure('duration.priority_resources', tags: %W(namespace:#{@namespace} sha:#{ENV['REVISION']})) do
```

Comment:
Absolutely, its done in https://github.com/Shopify/kubernetes-deploy/pull/118/files#diff-ae1efd633a69ca93e2ad9f56702555ebR142